pkg/pod/status.go

```diff
@@ -266,11 +279,30 @@ func createMessageFromResults(results []result.RunResult) (string, error) {
 	return string(bytes), nil
 }
 
+func findStepResultsFetchedByTask(stepName string, specResults []v1.TaskResult) (map[string]string, error) {
+	neededStepResults := map[string]string{}
+	for _, r := range specResults {
+		if r.Value != nil {
+			if r.Value.StringVal != "" {
+				sName, resultName, err := v1.ExtractStepResultName(r.Value.StringVal)
+				if err != nil {
+					return nil, err
+				}
+				if StepName(sName, 0) == stepName {
```

Comment:
why passing 0 here? Could sName be empty and it is not the first step?

Reply:
That's an artifact of using an existing function that takes an index. Here, it is meaningless so I just passed it `0`. I could also just append the prefix myself via `fmt.Sprintf`. I'll just do that to avoid confusion.

Reply:
👍

Reply:
done.